README.rst

```diff
@@ -137,6 +137,42 @@ or, clone the repository directly from GitHub:
   cd exoctk
   python setup.py [install|devlop]
 
+Obtain the ``exoctk`` Data
+--------------------------
+
+To obtain the ``exoctk`` data, visit the `ExoCTK website <https://exoctk.stsci.edu/>`_ and navigate to the "ExoCTK Data Download" tab. 
+This will start a download of a zipped package of all the data you need to run
+``exoctk``. 
+
+Because it is a fairly hefty data download, you'll need to be thoughtful about
+how you unzip it. 
+
+- For Mac OS, use the default application to unzip the file (Archive Utility or
+  similar depending on version.) 
+- For Windows, use the default application to unzip the file (Extract or
+  similar depending on version.)
+- For Linux, use the java unzip :
+
+::
+
+    jar -xf exoctk_data.zip
+
+- If you try to use ``unzip``, ``7zip``, or ``gzip``, the files will likely be corrupted. 
```

Comment:
For some odd reason, I see this line as split up onto two lines when viewing the file.

Reply:
![Screen Shot 2019-08-06 at 5 00 23 PM](https://user-images.githubusercontent.com/21206463/62576896-bfc6d080-b86b-11e9-9089-e63c6aa5e8d7.png)

This line appears as broken for you?

Reply:
Nevermind, it seemed to have fixed itself!